contracts/Governance/Timelock.sol

```diff
@@ -1,123 +1,120 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.13;
+pragma experimental ABIEncoderV2;
 
-contract Timelock {
-    event NewAdmin(address indexed newAdmin);
-    event NewPendingAdmin(address indexed newPendingAdmin);
-    event NewDelay(uint256 indexed newDelay);
-    event CancelTransaction(bytes32 indexed txHash, address indexed target, uint256 value, string signature, bytes data, uint256 eta);
-    event ExecuteTransaction(bytes32 indexed txHash, address indexed target, uint256 value, string signature, bytes data, uint256 eta);
-    event QueueTransaction(bytes32 indexed txHash, address indexed target, uint256 value, string signature, bytes data, uint256 eta);
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract Timelock is Ownable{
```

Comment:
clever solution, it looks really easy to usecan't wait for us to have it ready and deployed